src/label_studio_sdk/types/lse_task_serializer_for_annotators.py

```diff
@@ -42,6 +42,7 @@ class LseTaskSerializerForAnnotators(UncheckedBaseModel):
     predictions_results: typing.Optional[str] = None
     predictions_score: typing.Optional[float] = None
     reviews_rejected: typing.Optional[int] = None
+    state: typing.Optional[str] = None
```

Comment:
Where is this change coming from, and why is it only in the annotator task serializer, not the other task serializers?